features/configuration/fail_if_no_examples.feature

```diff
@@ -0,0 +1,46 @@
+Feature: fail if no examples
+
+  Use the `fail_if_no_examples` option to make RSpec exit with exit status 1
+  if there are 0 examples. Using this option, it is recommended to add:
+  a `--require spec_helper` option to `.rspec` file to ensure the `fail_if_no_examples`
+  option is set even if no spec files are loaded.
+
+  This option may be particularly useful when you happen to not run RSpec tests
+  locally, but rely on CI to do this. This prevents from false positive builds,
+  when you expected some RSpec examples to be run, but none were run. Such
+   a situation may be caused by your misconfiguration or regression/major changes
+  in RSpec.
```

Comment:
You hand-wrapped this narrative nicely, which would usually be what we want...but there's a rendering issue on relish.  It inserts line breaks wherever we have them in these paragraphs, which means that when it renders the lines don't look even since it's not using a fixed-width font like it renders here on github.  To make it look nice, we usually remove the hand-wrapping, making each paragraph one long line.

Do you mind doing that?